src/core/payments/constants.ts

```diff
@@ -51,3 +51,19 @@ export const BUFFER_DENOMINATOR = 10n
  */
 export const STORAGE_SCALE_MAX = 10_000_000
 export const STORAGE_SCALE_MAX_BI = BigInt(STORAGE_SCALE_MAX)
+
+/** PDP Leaf Size - the payment rate is based on `rawSize` bytes rounded up to the next multiple of 32.
+ *
+ * @see - https://github.com/FilOzone/synapse-sdk/issues/339#issue-3539254596
+ */
+export const PDP_LEAF_SIZE = 32
+
+/**
+ * Pad raw size to the next multiple of 32 bytes
+ *
+ * @param rawSizeBytes - The actual size in bytes
+ * @returns Padded size (next multiple of 32)
+ */
+export function padSizeToPDPLeaves(rawSizeBytes: number): number {
+  return Math.ceil(rawSizeBytes / PDP_LEAF_SIZE) * PDP_LEAF_SIZE
+}
```

Comment:
this shouldn't go in constants.ts it should probably go in a new file: `/core/payments/utils.ts`